tests/engine/test_calc_job.py

```diff
@@ -558,3 +573,31 @@ def parse_retrieved_output(_, __):
     result = process.parse()
     assert isinstance(result, ExitCode)
     assert result.status == final
+
+
+@pytest.mark.usefixtures('clear_database_before_test')
+def test_additional_retrieve_list(get_process, fixture_sandbox):
```

Comment:
So, since we are "merging" the `additional_retrieve_list` with the `retrieve_list`, would it perhaps be convenient to also check the case were you have content in both lists? Just to make sure there is no over-writting happening.

Reply:
good point. Will add a test.